bot/exts/evergreen/issues.py

```diff
@@ -180,7 +180,12 @@ async def issue(
     @commands.Cog.listener()
     async def on_message(self, message: discord.Message) -> None:
         """Command to retrieve issue(s) from a GitHub repository using automatic linking if matching <repo>#<issue>."""
-        if not(
+        # Ignore messages from DMs
+        if not message.guild:
+            return
```

Comment:
This should maybe send a message like riddle does? There's an argument there that it's a message, not a command, so it shouldn't error, but I don't know.

Reply:
No, this is a message argument, so we don't want to send messages on every message the user puts in the DMs.

Reply:
agreed, we don't want to send an error on every message in a DM, failing silently here is fine imo.

Reply:
If you move the regex up, you can get around this issue, and I think having an explicit “you can’t use issues in DM” is more useful than nothing. I can live without it.